autoload/probe/file.vim

```diff
@@ -72,6 +76,26 @@ function! probe#file#scan()
     return s:file_caches[s:hash]
 endfunction
 
+function! s:is_git()
+    let metadir_name = fnamemodify(s:find_metadir(), ':t')
+    return metadir_name == '.git'
+endfunction
+
+function! s:scan_git()
+    let files = []
+    echo "Scanning " . s:find_repo_root()
+
+    let cmd = "git ls-files; git submodule foreach -q 'for file in $(git ls-files); do echo \"$path/$file\"; done'"
```

Comment:
The `--other` flag makes `git ls-files` return the opposite of what I want (untracked and ignored files).

I assume you are suggesting that option because you want untracked (but not ignored) files to show up? 

I actually like excluding untracked files but it probably be nice to make that configurable.